### PR TITLE
Build new features page for /core

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new page at `/core/features`

## QA

- View the page at: https://ubuntu-com-9112.demos.haus/core/features
- Check against [copydoc](https://docs.google.com/document/d/10S6FGVRA1R1qi9v859CnFrl8rS25Za2SjEmSpPVQkzY/edit#heading=h.2rlj58ccsw9y)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes [#3666](https://github.com/canonical-web-and-design/web-squad/issues/3666)

### CODE REVIEW

**Comment by anasereijo** [comment]
- [ ] contact us section: make the illustration the same size as we have on https://ubuntu.com/kubernetes/features
- [ ] snap logo and ubuntu logo: width:120px
- [ ] IoT app stores illustration- height:180px
- [ ] waiting on the illustration for Snaps

**Comment by bethcollins92** [comment]
Fixed all design requests just waiting on snap illustration

**Comment by anasereijo** [comment]
please hide the contact us illustration on mobile

File: templates/core/features/index.html [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <h1>Ubuntu Core features</h1>
+    <p class="p-heading--4">What’s under the hood</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```
Comment (pmahnke) [inline templates/core/features/index.html:17]: extra div row

File: templates/core/features/index.html [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    <div class="row u-equal-height">
+      <div class="col-8">
+        <h2>Snaps</h2>
+        <p>Ubuntu Core is built from snaps, a secure, confined, dependency-free, cross-platform Linux packaging format. Snaps are entirely self-contained, even to the point of encapsulating their own file system. This means they include everything they need to run in any environment. They're used by Ubuntu Core to both compose the image that's run on a device, and to deliver consistent and reliable software updates, even to low-powered, inaccessible and remotely administered embedded and IoT systems.</p>
+        <p>
+          <a class="p-link--external" href="https://core.docs.ubuntu.com/en/coresnaps">Learn more about snaps</a>
```
Comment (pmahnke) [inline templates/core/features/index.html:23]:
```suggestion
          <a href="/core/docs/snaps-in-ubuntu-core">Learn more about snaps</a>
```

File: templates/core/features/index.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+            alt="",
+            width="167",
+            height="200",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+      </div>
+    </div>
+  </div>
```
Comment (pmahnke) [inline templates/core/features/index.html:194]: extra closing div?

**Review by pmahnke — APPROVED** [review]
A few inline comments.

File: templates/core/features/index.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+</section>
+
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Snaps</h2>
+      <p>Ubuntu Core is built from snaps, a secure, confined, dependency-free, cross-platform Linux packaging format. Snaps are entirely self-contained, even to the point of encapsulating their own file system. This means they include everything they need to run in any environment. They're used by Ubuntu Core to both compose the image that's run on a device, and to deliver consistent and reliable software updates, even to low-powered, inaccessible and remotely administered embedded and IoT systems.</p>
+      <p>
+        <a href="/core/docs/snaps-in-ubuntu-core">Learn more about snaps</a>
```
Comment (pmahnke) [inline templates/core/features/index.html:22]: needs the >

**Review by pmahnke — APPROVED** [review]
1 minor change... otherwise good... we will need to remember to add the links into this page as we create the other pages.

File: templates/core/features/index.html (hunk 4 above)
Comment (bethcollins92) [reply templates/core/features/index.html:22]: Great thanks, yeah I'll take note of where we will need to add them

**Pull request merged**